Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.yml

```diff
@@ -2324,6 +2328,364 @@ script:
       name: lastUpdate
     description: Gets the list of incidents that were modified since the last update time. Note that this method is here for debugging purposes. The get-modified-remote-data command is used as part of a Mirroring feature, which is available in Cortex XSOAR from version 6.1.
     name: get-modified-remote-data
+  - name: bmc-itsm-support-group-list
+    description: Lists all support groups. Useful for getting possible (Company, Support Organization, Support Group) triplets.
+    arguments:
+    - name: limit
+      description: The maximum number of records to retrieve.
+      type: String
+      defaultValue: "50"
+      predefined:
+      - ""
+    - name: page_size
+      description: The maximum number of records to retrieve per page.
+      type: String
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: page
+      description: The page number of the results to retrieve.
+      type: String
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: company
+      description: Company name. Used as a filtering argument.
+      type: String
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: support_organization
+      description: Support organization name. Used as a filtering argument.
+      type: String
+      defaultValue: ""
+      predefined:
+      - ""
+    - name: support_group
+      description: Support group name. Used as a filtering argument.
+      type: String
+      defaultValue: ""
+      predefined:
+      - ""
+    outputs:
+    - type: String
+      contextPath: BmcITSM.SupportGroup.Support Group ID
```

Comment:
did you double check these are clickable in the xsoar ui? I wasnt aware before that we support spacing here
@meital-aizen fyi, do we want to change the context at all? Usually we return it as it came from the api

Reply:
Yes, they are clicable

Reply:
@JudahSchwartz is correct, we save to the context whatever is returned from the API response - I assume it's not with spacing. @dberezovik  Please fix this

Reply:
@meital-aizen , I can remove spacing - no issues here, but API really does return the values with spaces:
![image](https://github.com/demisto/content/assets/103488527/c2aaf30f-18a0-4851-b0e7-84e57939a293)

Should I rename it to SupportGroupID ?

Reply:
@dberezovik Please remove spacing from all the fields - it might cause problems if not (if i'm not mistaken)

Reply:
@meital-aizen , ok, will do it first thing on Monday. Thanks

Reply:
Done: spaces removed